app/src/main/java/com/android/partagix/model/ItemViewModel.kt

```diff
@@ -53,7 +74,8 @@ class ItemViewModel(
   fun saveWithUiState() {
     if (_uiState.value.item.id == "") {
       database.createItem(
-          "Yp5cetHh3nLGMsjYY4q9" /* todo get the correct userId */, _uiState.value.item)
+          "Yp5cetHh3nLGMsjYY4q9" /* todo get the correct userId */,
```

Comment:
This should not be too hard to do using : 

`user = FirebaseAuth.getInstance().currentUser?.uid`